app/assets/stylesheets/spotlight/_header.scss

```diff
@@ -31,112 +28,93 @@ $masthead-image-blur: 1px;
-  @include transparent-masthead-navigation-menu();
-  background-color: $navbar-transparent-bg;
-  border-top: 1px solid $navbar-transparent-border;
-}
-
-.navbar + .navbar {
-  margin-top: 0;
-}
-
 #header-navbar {
   margin-bottom: 0;
 }
 
-#exhibit-masthead {
-  height: $masthead-height;
-  margin-bottom: -50px;
+.masthead {
+  // allow dropdowns in the top nav to stack above the masthead
+  z-index: $zindex-navbar - 1;
+  margin-bottom: $padding-large-vertical * 2.5;
+  border-bottom: 1px solid $navbar-default-border;
+
+  background-color: $masthead-bg;
   padding: 0;
-  position: static;
+  position: relative;
+  overflow: hidden;
 
-  & > .container {
+  .site-title {
     position: relative;
-    top: 33%;
-    -webkit-transform: translateY(-50%);
-    transform: translateY(-50%);
+    white-space: nowrap;
+    padding-bottom: $padding-base-vertical;
   }
 
-  &.with-page-masthead {
-    margin-bottom: $padding-base-vertical;
-    margin-top: -51px;
+  small {
+    display: block;
+    margin-top: -$padding-base-vertical;
+    padding-top: $padding-base-vertical;
+    padding-bottom: $padding-base-vertical;
+    @extend .hidden-xs;
+  }
 
-    > .container {
-      top: 60%;
+  .navbar {    
+    &:last-child {
+      margin-bottom: 0;
     }
+  }
 
-    .navbar-brand {
-      font-size: $font-size-large;
-      text-shadow: 1px 1px 0 $black;
-    }
+  .navbar-form {
+    text-align: right;
+  }
+}
 
-    .search-title {
-      text-align: center;
+.resource-masthead {
+  .site-title {
+    @extend .text-center;
+  }
 
-      small {
-        text-transform: uppercase;
-      }
-    }
+  .navbar {
+    border-bottom: 1px solid $navbar-transparent-border;
+    border-top: 0;
+    margin-bottom: $padding-small-vertical;
   }
 
-  .site-title {
-    font-size: 30px;
+  .navbar-nav {
+    float: right;
   }
 
-  small {
-    display: block;
-    padding-top: $padding-base-vertical;
-    @extend .hidden-xs;
+  .search-query-form {
+    display: none;
   }
+}
 
-  &.with-image {
-    .site-title,
-    .search-title {
-      color: $white;
-      text-shadow: 1px 1px 0 $black;
-    }
+.site-title-container {
+  max-height: $masthead-height - $navbar-height - 2;
+  
+  padding-top: $padding-large-vertical;
+  padding-bottom: $padding-large-vertical;
+}
 
-    small {
-      color: $white;
-    }
+.image-masthead {
+  .navbar {
+    @include transparent-masthead-navigation-menu();
+    background-color: $navbar-transparent-page-bg;
   }
 
-  &.with-page-masthead {
-    .site-title {
-      margin-top: $padding-large-vertical * 3;
-    }
+  .h1 {
+    color: $white;
+    text-shadow: 1px 1px 0 $black;
+  }
+
+  small {
+    color: $white;
   }
 
   // This is to add a background image to the masthead, in a way that
   // enables the site title and subtitle text to be visible above it.
   .background-container {
     @include masthead-background-containers();
+    height: $masthead-height;
```

Comment:
Properties should be ordered background-repeat, background-size, border-right, -webkit-filter, filter, height, margin-left, margin-top, overflow